parsers/eiffel.c

```diff
@@ -149,6 +149,7 @@ static const keywordTable EiffelKeywordTable [] = {
 	{ "indexing",       KEYWORD_indexing   },
 	{ "infix",          KEYWORD_infix      },
 	{ "inherit",        KEYWORD_inherit    },
+	{ "insert",         KEYWORD_insert     },
```

Comment:
if `insert` is an alias of `inherit`, you probably should rather simply map `"insert"` to `KEYWORD_inherit` and not modify anything else